dct_backend/dct.py

```diff
@@ -30,3 +34,54 @@ def save_food_item(self, name: str, station: str, meal_time: str,
         item = FoodItem(name, station, meal_time, calories, categories)
         self.menu.append(item)
         return item
+
+    """
+        incoming_dct_traffic adds people who are scanning into the dct
+
+        takeout: int
+            The current number of takeout scanning in
+
+        dinein: int
+            The current number of dinein scanning in
+
+        returns: Capacity
+            The total number of takeout, dineine, and total capacity
+            of takeout + dinein
+    """
+    def incoming_dct_traffic(self, takeout: int, dinein: int) -> Capacity:
```

Comment:
I don't think it is a good idea returning a `Capacity` object. It gives the impression that the `Capacity` object returned can be manipulated and still be valid.

I would recommend just returning the 3 variables:
`return self.takeout_capacity, self.dinein_capacity, self.total_capacity`